### ISSUE TITLE
"No potentially focusable controls found in modal dialog" logged to JavaScript console

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7638]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.4.731
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- start IDE, open Chrome(ium) dev tools, show JS console
- Tools / Project Options (if you have a project open) or Help / Accessibility / Accessibility Options

### Describe the problem in detail
- Warning logged from the front-end in JS console: "WARNING: No potentially focusable controls found in modal dialog"
- Despite this, if you tap shift, you can see that focus was indeed placed in the right location

### Describe the behavior you expected
- This message shouldn't be shown unless we are actually in a situation where a dialog has no keyboard-focusable elements
- Note I have also seen this, sometimes, with Tools / Global Options, but not always

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by R-Hannibal** [comment]
@gtritchie - This issue is marked as `Desktop or Server`, but is there a way to verify on Desktop (and if so, how do you access the Chromium devtools > JS Console on desktop?)? I thought this was referring to browser console logs (Chrome browser > F12 > Console), but want to make sure I'm not missing something.

**Comment by gtritchie** [comment]
Yes, you can get at the Chromium dev tools within RStudio Desktop. Just right-click blank area of toolbar and select "Inspect Element".

<img width="433" alt="screenshot of RStudio Desktop showing right-click menu on blank area of toolbar" src="https://user-images.githubusercontent.com/10569626/91090635-3d41b800-e60a-11ea-91ad-c3a0e4b91e0f.png">

**Comment by R-Hannibal** [comment]
Verified Fixed on: RStudio Version - 1.4.734, R Version - 4.0.2, Windows 10 x64 - Desktop 

Spurious console Warning for `No potentially focusable controls found in modal dialog.` is no longer being emitted in the JS Console. Comparison tested between 1.4.706 and current Daily using both project options and accessibility options, warning is thrown on the old version, not present on the new build.
